src/component/Button.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/button-has-type */
+import React from 'react';
+
+export default function Button(props) {
+  return (
+    <button className="">{props.name}</button>
+  );
+}
```

Comment:
`name` prop should be a string, enforce this with STRINGS [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png)]